src/App.tsx

```diff
@@ -1,18 +1,63 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState } from 'react';
+
+import { Todo } from './types/Todo';
+import { TodosFilter } from './components/TodosFilter';
+import { TodoList } from './components/TodoList';
 
 export const App: React.FC = () => {
+  const [inputText, setInputText] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isFilter, setIsFilter] = useState('');
+
+  const generateId = (): number => {
+    return +new Date();
+  };
+
+  const handleAddTodo = (event:React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
use a more descriptive handler name. It is basically not adding new todo. it is just title change
```suggestion
  const handleInputChange = (event:React.ChangeEvent<HTMLInputElement>) => {
```